No such file or directory on builder-cli android

**Describe the bug**
Revanced Builder doesnt create revanced.apk 

**To Reproduce**
Steps to reproduce the behavior:

1. Execute node . and do anything like a normal user would do, but on android
2. error appears at end of compiling

**Screenshots**

![20220725_174154.jpg](https://user-images.githubusercontent.com/94485936/180886636-42ef10d8-79c6-429d-be54-dabf73607856.jpg)

**revanced-builder Version:v1.0.0**

**Desktop (please complete the following information):**

- OS: Android, One Ui 2
- Version: 10

@reisxd So, I tried to manually run the `java -jar .....` command, and this is what I get towards the end:

```
INFO: Compiling resources
WARNING: error: resource 'drawable/$$avd_anim__1__0' has invalid entry name '$$avd_anim__1__0.
WARNING: error: resource 'drawable/$$avd_anim__1__1' has invalid entry name '$$avd_anim__1__1.
WARNING: error: resource 'drawable/$$avd_anim__2__0' has invalid entry name '$$avd_anim__2__0.
WARNING: error: resource 'drawable/$$avd_anim__2__1' has invalid entry name '$$avd_anim__2__1.
WARNING: error: resource 'drawable/$$avd_anim__3__0' has invalid entry name '$$avd_anim__3__0.
WARNING: error: resource 'drawable/$$avd_anim__3__1' has invalid entry name '$$avd_anim__3__1.
WARNING: error: resource 'drawable/$$sud_fourcolor_progress_bar__10__0' has invalid entry name '$$sud_fourcolor_progress_bar__10__0.
WARNING: error: resource 'drawable/$$sud_fourcolor_progress_bar__11__0' has invalid entry name '$$sud_fourcolor_progress_bar__11__0.
WARNING: error: resource 'drawable/$$sud_fourcolor_progress_bar__12__0' has invalid entry name '$$sud_fourcolor_progress_bar__12__0.
WARNING: error: resource 'drawable/$$sud_fourcolor_progress_bar__1__0' has invalid entry name '$$sud_fourcolor_progress_bar__1__0.
WARNING: error: resource 'drawable/$$sud_fourcolor_progress_bar__2__0' has invalid entry name '$$sud_fourcolor_progress_bar__2__0.
WARNING: error: resource 'drawable/$$sud_fourcolor_progress_bar__3__0' has invalid entry name '$$sud_fourcolor_progress_bar__3__0.
WARNING: error: resource 'drawable/$$sud_fourcolor_progress_bar__4__0' has invalid entry name '$$sud_fourcolor_progress_bar__4__0.
WARNING: error: resource 'drawable/$$sud_fourcolor_progress_bar__5__0' has invalid entry name '$$sud_fourcolor_progress_bar__5__0.
WARNING: error: resource 'drawable/$$sud_fourcolor_progress_bar__6__0' has invalid entry name '$$sud_fourcolor_progress_bar__6__0.
WARNING: error: resource 'drawable/$$sud_fourcolor_progress_bar__7__0' has invalid entry name '$$sud_fourcolor_progress_bar__7__0.
WARNING: error: resource 'drawable/$$sud_fourcolor_progress_bar__8__0' has invalid entry name '$$sud_fourcolor_progress_bar__8__0.
WARNING: error: resource 'drawable/$$sud_fourcolor_progress_bar__9__0' has invalid entry name '$$sud_fourcolor_progress_bar__9__0.
WARNING: error: resource 'drawable/$avd_anim__0' has invalid entry name '$avd_anim__0.
WARNING: error: resource 'drawable/$avd_anim__1' has invalid entry name '$avd_anim__1.
brut.androlib.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [/data/data/com.termux/files/usr/bin/aapt2, compile, --dir, /data/data/com.termux/files/home/revanced-builder/revanced-cache/res, --legacy, -o, /data/data/com.termux/files/home/revanced-builder/revanced-cache/build/resources.zip]
        at brut.androlib.res.AndrolibResources.aapt2Package(AndrolibResources.java:372)
        at brut.androlib.res.AndrolibResources.aaptPackage(AndrolibResources.java:626)
        at app.revanced.patcher.Patcher.save(Patcher.kt:199)
        at app.revanced.cli.patcher.Patcher.start$revanced_cli(Patcher.kt:28)
        at app.revanced.cli.command.MainCommand.run(MainCommand.kt:159)
        at picocli.CommandLine.executeUserObject(CommandLine.java:1939)
        at picocli.CommandLine.access$1300(CommandLine.java:145)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2314)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
        at picocli.CommandLine.execute(CommandLine.java:2078)
        at app.revanced.cli.main.MainKt.main(Main.kt:7)
Caused by: brut.common.BrutException: could not exec (exit code = 1): [/data/data/com.termux/files/usr/bin/aapt2, compile, --dir, /data/data/com.termux/files/home/revanced-builder/revanced-cache/res, --legacy, -o, /data/data/com.termux/files/home/revanced-builder/revanced-cache/build/resources.zip]
        at brut.util.OS.exec(OS.java:99)
        at brut.androlib.res.AndrolibResources.aapt2Package(AndrolibResources.java:368)
        ... 13 more
```
And when I run the `aapt2` command, I get the same thing, except it's not a WARNING, but an error.

--- a/index.js
+++ b/index.js
@@ -377,44 +377,28 @@ async function androidBuild () {
     }
   }
 
-  try {
-    await actualExec('aapt2');
-  } catch (e) {
-    if (!e.stderr.includes('not found')) {
-      console.log('aapt2 is already installed.');
-    } else {
-      console.log("Couldn't find aapt2, installing...");
-      await dloadFromURL(
-        'https://github.com/reisxd/revanced-cli-termux/raw/main/aapt2.zip',
-        'aapt2.zip'
-      );
-      console.log(`The architecture is ${os.arch()}`);
-      await actualExec('unzip aapt2.zip');
-      switch (os.arch()) {
-        case 'arm64': {
-          await actualExec(
-            'cp arm64-v8a/aapt2 /data/data/com.termux/files/usr/bin/aapt2'
-          );
-          await actualExec(
-            'chmod +x /data/data/com.termux/files/usr/bin/aapt2'
-          );
-          break;
-        }
+  console.log('Installing aapt2');
+  await dloadFromURL(
+    'https://github.com/reisxd/revanced-cli-termux/raw/main/aapt2.zip',
+    'aapt2.zip'
+  );
+  console.log(`The architecture is ${os.arch()}`);
+  await actualExec('unzip aapt2.zip');
+  switch (os.arch()) {
+    case 'arm64': {
+      await actualExec('cp arm64-v8a/aapt2 revanced/aapt2');
+      await actualExec('chmod +x revanced/aapt2');
+      break;
+    }
 
-        case 'arm': {
-          await actualExec(
-            'cp armeabi-v7a/aapt2 /data/data/com.termux/files/usr/bin/aapt2'
-          );
-          await actualExec(
-            'chmod +x /data/data/com.termux/files/usr/bin/aapt2'
-          );
-          break;
-        }
-      }
-
-      console.log('aapt2 has been installed.');
+    case 'arm': {
+      await actualExec('cp armeabi-v7a/aapt2 revanced/aapt2');
+      await actualExec('chmod +x revanced/aapt2');
+      break;
     }
   }
+
+  console.log('aapt2 has been installed.');
 }
 
 (async () => {
@@ -611,7 +595,7 @@ async function androidBuild () {
       const { stdout, stderr } = await actualExec(
         `java -jar ${jarNames.cli} -b ${jarNames.patchesJar} ${
           os.platform() === 'android'
-            ? '--custom-aapt2-binary /data/data/com.termux/files/usr/bin/aapt2'
+            ? '--custom-aapt2-binary revanced/aapt2'
             : ''
         } -t ./revanced-cache --experimental -a ./revanced/youtube.apk ${
           jarNames.deviceId

